### ISSUE TITLE
Feature Request: optionally avoid preventDefault for events

### ISSUE THREAD

**Issue by montaga** [CindyJS/CindyJS#803]
When developing applets for mobile devices, the behavior of CindyJS to `steal` all pointer events often is problematic. This can be traced back to the `preventDefault()`s in `Event.js`.
Sometimes it is desirable to be able to disable this behavior. In many cases it is a quick fix to add `style="pointer-events: none;"` to the CSCanvas, however, often it is desirable to be able to both click on the applet and scroll the applet.

Could we introduce a global parameter `preventDefaultEvents`, which per default is set to `true`, but which can be set to `false`, to have this behavior?

**Comment by strobelm** [comment]
Yes, this would be great! Perhaps we could set the default value dependent on the device type.

**Comment by kortenkamp** [comment]
Closing this issue due to inactivity.
